README.md

```diff
@@ -70,25 +70,34 @@
 </details>
 
 Web/Desktop [try out the web build!](https://DonutWare.github.io/Fladder)
-#### Notice: The github hosted web build only allows https connections. This is a limitation on githubs end, for self-hosted versions http works fine. 
+#### Notice: The GitHub hosted web build only allows `https` connections. This is a limitation on GitHub's end, for self-hosted versions `http` works fine. 
 
 ## Planned Features
 
 * Music playback
-* Android TV
+* Android TV support
 * Server management
 * And more???
 
-## Download
+## Installation
 
-The latest release executeables for Windows, MacOS, iOS, Android and Web can be found [here](https://github.com/DonutWare/Fladder/releases).
+The latest executables for Windows, macOS, Linux, iOS and Android can be found on the [releases page](https://github.com/DonutWare/Fladder/releases).
 
-### Playstore
-The Android app is currently in testing you can help testing by joining this group https://groups.google.com/g/fladder
```

Comment:
Lets try to keep this part short. This list would become very long if we wrote instructions for all combinations of packge manager, distro etc.

An alternative might be to create a second readme specific for linux. But we could also just point to the need for libmpv when users use the binary version.